plugins/kubernetes/app/views/admin/kubernetes/deploy_group_roles/_form.html.erb

```diff
@@ -12,7 +12,8 @@
     <div class="form-group">
       <%= form.label :project_id, 'Project', class: "col-lg-2 control-label" %>
       <div class="col-lg-2">
-        <%= form.select :project_id, current_user.administrated_projects.pluck(:name, :id), {}, class: "form-control", disabled: @deploy_group_role.persisted? %>
+        <% projects = current_user.administrated_projects.where(id: Kubernetes::Role.pluck(:project_id)).pluck(:name, :id) %>
```

Comment:
How about creating a scope or class-level method on Project, so you can say `Project.with_kubernetes_roles` or something like that?